charts/openshift-metering/templates/openshift-reporting/report-queries/pod-cpu.yaml

```diff
@@ -365,19 +365,18 @@ spec:
     SELECT
       timestamp '{| default .Report.ReportingStart .Report.Inputs.ReportingStart| prestoTimestamp |}' AS period_start,
       timestamp '{| default .Report.ReportingEnd .Report.Inputs.ReportingEnd | prestoTimestamp |}' AS period_end,
-      usage.namespace,
+      usage.namespace as usage_namespace,
+      request.namespace as request_namespace,
```

Comment:
We can remove this as the usage and request namespaces are going to be the same.

Reply:
Fixed